packages/grafana-ui/src/types/panel.ts

```diff
@@ -16,6 +16,8 @@ export interface PanelData {
   series: DataFrame[];
   request?: DataQueryRequest;
   error?: DataQueryError;
+  // Contains the shifted time range if a request uses relative time
+  timeRange?: TimeRange;
```

Comment:
Not certain about naming here: `newTimeRange`, `shiftedTimeRange` or `updatedTimeRange` have also crossed my path.